### ISSUE TITLE
Not compatible with recent releases of mochoa

### ISSUE THREAD

**Issue by IdanAdar** [stanleyhlng/mocha-multi-reporters#57]
@stanleyhlng  

Previously this worked:

```
"scripts": {
    "test": "jenkins-mocha --recursive --reporter mocha-multi-reporters --reporter-options configFile=config/mocha-config.json --exit"
  },
```

However now it is failing with:

```
"--reporter <reporter>" can only be specified once
```

**Comment by pkuczynski** [comment]
I think this is not valid anymore?

**Comment by francisu** [comment]
Fixed in 1.5.1
